girder/molecules/molecules/avogadro.py

```diff
@@ -1,34 +1,61 @@
-from avogadro.core import *
-from avogadro.io import *
 import json
+import requests
+
+from avogadro.core import Molecule
+from avogadro.io import FileFormatManager
+
+from girder.models.setting import Setting
+
 from jsonpath_rw import parse
 
+from molecules.constants import PluginSettings
+
+
+def avogadro_base_url():
+    base_url = Setting().get(PluginSettings.AVOGADRO_BASE_URL)
+    if base_url is None:
+        base_url = 'http://localhost:5000'
+
+    return base_url
+
+
 def convert_str(str_data, in_format, out_format):
     mol = Molecule()
     conv = FileFormatManager()
     conv.read_string(mol, str_data, in_format)
 
     return conv.write_string(mol, out_format)
 
+
 def atom_count(str_data, in_format):
-    mol = Molecule()
-    conv = FileFormatManager()
-    conv.read_string(mol, str_data, in_format)
+    base_url = avogadro_base_url()
+    path = 'properties'
+    url = '/'.join([base_url, path, 'atom'])
+
+    data = {
+        'format': in_format,
+        'data': str_data,
+    }
+
+    r = requests.post(url, json=data)
```

Comment:
We should add a [raise_for_status](https://2.python-requests.org/en/master/api/#requests.Response.raise_for_status), so we can see when the requests fails.